### ISSUE TITLE
3.0.0-beta.2 - Permission podspecs not included in react-native-permissions node_module

### ISSUE THREAD

**Issue by dylan-westbury** [zoontek/react-native-permissions#547]
## Summary

3.0.0-beta.2 not generating podspec files for ios permission. Is there a new way to install them? pod install throws error

`[!] No podspec found for `Permission-Notifications` in `../node_modules/react-native-permissions/ios/Notifications.podspec`

<img width="1018" alt="Screen Shot 2020-11-28 at 5 53 22 pm" src="https://user-images.githubusercontent.com/16129043/100495834-62af0680-31a3-11eb-93ca-97644486eb09.png">

## Environment info

```
System:
    OS: macOS 10.15.7
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 938.26 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.16.3 - /usr/local/bin/node
    Yarn: 1.19.1 - ~/.yarn/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.2, DriverKit 20.0, macOS 11.0, tvOS 14.2, watchOS 7.1
    Android SDK:
      API Levels: 22, 23, 25, 26, 27, 28, 29
      Build Tools: 26.0.3, 27.0.3, 28.0.3, 29.0.2
      System Images: android-26 | Google Play Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
      Android NDK: 20.0.5594570
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5900203
    Xcode: 12.2/12B45b - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_181 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.3 => 0.63.3 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

```

Library version: 3.0.0-beta.2

## Steps to reproduce
- Explain the steps we need to take to reproduce the issue:
1. yarn add react-native-permissions (3.0.0 beta is installed but missing permissions podspecs within npm modules ios folder)
2. pod install (gives error [!] No podspec found for `Permission-Notifications` in `../node_modules/react-native-permissions/ios/Notifications.podspec`)

Describe what you expected to happen:
1. podspec files should install or new method to add to pod file should be updated in documentation

**Comment by zoontek** [comment]
They are in each handler directory now, which allow us to remove the ".podspec".

Read the 3.0.0 documentation: https://github.com/zoontek/react-native-permissions/tree/3.0.0#ios

**Comment by zoontek** [comment]
Update: I published the `3.0.0` release. The documentation on master is now the `3.0.0` one.
